docs/adrs/0081-use-asdf-to-manage-node-and-golang-versions-in-development.md

```diff
@@ -0,0 +1,61 @@
+---
+title: 0081 Use ASDF to Manage Node and Golang versions in Development
+---
+
+# Use ASDF to Manage Node and Golang versions in Development
+
+:::info
+
+This ADR supersedes [ADR-0046](0046-use-nodenv.md) and [ADR-0056](0056-use-asdf-to-manage-golang-versions-in-development.md)
+
+:::
+
+## Considered Alternatives
+
+* Do nothing, keep using asdf for golang and nodenv for node
+* Use asdf to manage golang and node versions
+
+## Decision Outcome
```

Comment:
**questions:** 

Do we need to mention somewhere here that going forward folks will use asdf? (i.e. if you have been using nodenv in the past, you're still welcome to use nodenv, but new folks will use asdf)

Also, do we want to have a ticket or something else to revisit migrating everyone over? I think getting everyone to use asdf would be useful for being able to help each other.

Other side question. How do folks migrate to using allowing asdf to manage their node version and go version?

Reply:
I added a quick guide for switching from Nodenv to asdf: 

https://github.com/transcom/mymove-docs/pull/386/commits/437c55c804874fb69c47e8dcbb4e6e3c49f14b57

Reply:
Once merged, I do plan on putting out an announcement on slack, and volunteer to help anyone who tries to switch and gets stuck

I think a ticket in the backlog to switch everyone over and then remove nodenv related docs once and for all is a good idea